browser/ui/brave_browser.cc

```diff
@@ -75,6 +97,29 @@ BraveBrowser::BraveBrowser(const CreateParams& params) : Browser(params) {
 
 BraveBrowser::~BraveBrowser() = default;
 
+bool BraveBrowser::ShouldShowBookmarkBar() const {
+  content::WebContents* web_contents = tab_strip_model_->GetActiveWebContents();
```

Comment:
this needs some explanation of what we are doing here and why it's different from upstream

Reply:
@simonhong Could you make a follow up issue (or feel free to push a commit with comments if simpler)?

Reply:
Added comment for this method - https://github.com/brave/brave-core/pull/28944/commits/41434f6dd1b1ab67987b36769632fbca3a5a016b